zerver/lib/zulip_update_announcements.py

```diff
@@ -0,0 +1,212 @@
+from dataclasses import dataclass
+from datetime import timedelta
+from typing import List, Optional
+
+from django.conf import settings
+from django.db import transaction
+from django.db.models import Q, QuerySet
+from django.utils.timezone import now as timezone_now
+from django.utils.translation import override as override_language
+
+from zerver.actions.message_send import (
+    do_send_messages,
+    internal_prep_huddle_message,
+    internal_prep_stream_message,
+)
+from zerver.lib.message import SendMessageRequest
+from zerver.models.realm_audit_logs import RealmAuditLog
+from zerver.models.realms import Realm
+from zerver.models.users import UserProfile, get_system_bot
+
+
+@dataclass
+class ZulipUpdateAnnouncement:
+    level: int
+    message: str
+
+
+# We don't translate the announcement message because they are quite unlikely to be
+# translated during the time between when we draft them and when they are published.
+zulip_update_announcements: List[ZulipUpdateAnnouncement] = [
+    ZulipUpdateAnnouncement(
+        level=1,
+        message=(
+            "This topic will receive [automated updates]({zulip_update_announcements_help_url}) "
+            "about new Zulip features.\n"
+            "\n"
+            "Organization administrators can [configure]({settings_url}) a different stream "
+            "for these updates (and [move]({move_content_help_url}) the initial updates there), "
+            "or [turn this feature off]({settings_url}) altogether."
+        ).format(
```

Comment:
Can we redo all these strings using the `"""` Python quote blocks? It'll be a lot more convenient to edit in that form.

Reply:
Done.